### PR TITLE
Dalia pr

### PR DESCRIPTION
**Pull request by DaliaAmr**
This JavaScript code implements a todo list application using the MVC (Model-View-Controller) pattern. The Model manages the data, including adding, updating, and deleting todos, while maintaining a unique ID for each todo. The View handles the user interface, rendering the todos and updating the display based on user interactions. The Controller facilitates the interaction between the Model and the View, initializing the application, and handling user input, such as adding new todos. This separation of concerns enhances code modularity, maintainability, and clarity.

### CODE REVIEW

**Review by Rubazy — APPROVED** [review]
-README File in line 1 the updates in the #Internship To-Do App (dina), and you're DaliaAmr.
-There are comments at every step, not the whole function or block of code.

**Review by nourannnnnnnnnnnn — APPROVED** [review]
well ,good

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Internship To-Do App
+# Internship To-Do App (dina)
```
Comment (omrmhd5) [inline README.md]: You are Dalia Not Dina

File: css/styles.css [hunk 2]
```diff
@@ -1,20 +1,164 @@
-* {
+body {
+    font-family: 'Poppins', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: ce=nter;
+    height: 100vh;
     margin: 0;
+    background-color: #6a0dad; /* Purple background */
```
Comment (omrmhd5) [inline css/styles.css:9]: Not Needed Comments

**Review by omrmhd5 — APPROVED** [review]
Overall Good Work but a lot of useless comments

File: index.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    },
+    {
+      id: 3,
+      title: "task 3",
+      completed: false,
+      created_at: new Date().toISOString(),
+      updated_at: new Date().toISOString(),
+    },
+  ],
+  nextId: 4, // Next ID to be used
```
Comment (DinaAyman03) [inline index.js:25]: In order to add a task after deleting all of the tasks, i recommend using this line of code :   model.addTodo({
        id: (model.getTodos()[model.getTodos().length - 1]?.id ?? 0) + 1,
        title: inputElemVal,
        completed: false,
        created_at: "",
        updated_at: "",
      });

**Pull request merged**